storage/statedb/cache_fastcache.go

```diff
@@ -52,30 +62,64 @@ func NewFastCache(config *TrieNodeCacheConfig) TrieNodeCache {
 		return nil
 	}
 
-	fc := &FastCache{fast: fastcache.LoadFromFileOrNew(config.FastCacheFileDir, config.LocalCacheSizeMB*1024*1024)} // Convert MB to Byte
-	stats := fc.UpdateStats().(fastcache.Stats)
+	fc := &FastCache{new: fastcache.New(config.LocalCacheSizeMB * 1024 * 1024)} // Convert MB to Byte
+	go fc.loadFromBackground(config)
 
-	logger.Info("Initialize local trie node cache (fastCache)",
-		"MaxMB", config.LocalCacheSizeMB, "FilePath", config.FastCacheFileDir,
-		"LoadedBytes", stats.BytesSize, "LoadedEntries", stats.EntriesCount)
+	fmt.Println("Initialize local trie node cache (fastCache)",
+		"MaxMB", config.LocalCacheSizeMB, "FilePath", config.FastCacheFileDir)
 	return fc
 }
 
+func (cache *FastCache) loadFromBackground(config *TrieNodeCacheConfig) {
+	start := time.Now()
+	cache.old = fastcache.LoadFromFileOrNew(config.FastCacheFileDir, config.LocalCacheSizeMB*1024*1024)
```

Comment:
And setting `cache.old` is done here, which is also processed in another goroutine. Therefore, read and write of `cache.old` is done in concurrent manner and seems to be a definite data race condition. 

> Unsynchronized, concurrent access to any variable from multiple goroutines where at least one of them is a write is undefined behavior by The Go Memory Model.

https://stackoverflow.com/a/41407827
https://golang.org/ref/mem